apps/opik-backend/src/main/java/com/comet/opik/domain/cost/CostService.java

```diff
@@ -108,4 +31,29 @@ public static BigDecimal calculateCost(@Nullable String rawModelName, @Nullable
 
         return modelPrice.calculator().apply(modelPrice, Optional.ofNullable(usage).orElse(Map.of()));
     }
+
+    private static void parseModelPrices() {
+        JsonNode pricesJson = JsonUtils.readJsonFile(PRICES_FILE);
+        pricesJson.fields().forEachRemaining(modelEntry -> {
+            String modelName = modelEntry.getKey();
+            JsonNode modelData = modelEntry.getValue();
+            String provider = Optional.ofNullable(modelData.get("litellm_provider"))
+                    .map(JsonNode::asText).orElse("");
+            if (!modelName.startsWith("ft:") && providers.contains(provider)) {
```

Comment:
This looks very error-prone and fragile in terms of API integration. Can we group the models with similar calculation logic and try to match it based on properties and models?